client/src/model/states/model/demographic_data.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Information about demographics at geo-granularity.
+ */
+export interface DemographicData {
+  geoId?: string;
```

Comment:
Can you comment as to what this might be? i.e. explain this is census geo id of a zip, county, state

Reply:
Done!